### ISSUE TITLE
Nextcloud Bad Request

### ISSUE THREAD

**Issue by masterkrieger** [LibrePhotos/librephotos#289]
## 🐛 Bug Report

* [x] I am including my log files in zip format.
* [x] I have looked for similar issues among all the former issues including closed ones.

### What Operating system and version is LibrePhotos running on:
Ubuntu 18.04

### What architecture is LibrePhotos running on:
x64

### How is LibrePhotos installed:
Docker

### If running via Docker or Kubernetes please list version including docker-compose:
docker Version: 19.03.13
docker-compose version: '3.8'

### Are you running LibrePhotos on a virtual machine if so please list:
Proxmox VM - Ubuntu 18.04

### How is you picture library mounted on the host (or in the virtual machine):
Local file system (VM)

### Description of issue:
Nextcloud Scan Directory (green) in `Dashboard -> Library`, but in Admin Area `/data` is the only Scan Directory option. Because there's no Nextcloud directories, the "Scan photos (Nextcloud)" button is not selectable in Library.

### How can we reproduce it:
Install Docker images from Docker-compose method instructions on docs (https://docs.librephotos.com/1/standard_install/). Updated admin password, `httpPort=3002`, and `timeZone=America/New_York` in ".env" file. Then ran `docker-compose up -d` After logging into LibrePhotos, I entered the Nextcloud credentials with a new Nextcloud App Password and the Nextcloud Scan Directory status turned green. In the Admin Area, I try adding the scan directory and only option is `/data`. I don't see any other directories.

#### Additional Information:

* Include a ZIP file containing your `log` files.
* Provide pictures or links to videos that clearly demonstrate the issue.
![image_2021-07-02_102602](https://user-images.githubusercontent.com/1848365/124289072-e6591900-db1f-11eb-9bc1-944560c2d3c1.png)
![image_2021-07-02_102617](https://user-images.githubusercontent.com/1848365/124289102-eeb15400-db1f-11eb-9cd1-c00faafabc04.png)
[librephotos_docker_logs.zip](https://github.com/LibrePhotos/librephotos/files/6755232/librephotos_docker_logs.zip)

**Comment by StarkZarn** [comment]
I'll also add that I'm seeing a similar issue when trying to _change_ a nextcloud scan directory. It worked properly the first time around and imported the photos. Now I'd like to move it a directory higher to grab all the photos. When I click the "update" button it closes the dialog but the directory doesn't update. This coincides with a log event in the backend container of:
```
Bad Request: /api/user/2/
```

I've tried starting a new session as that user as well to no avail. Seems potentially related to this.

**Comment by masterkrieger** [comment]
I've re-installed using the method I mentioned above:

Install Docker images from Docker-compose method instructions on docs (https://docs.librephotos.com/1/standard_install/). Updated admin password, httpPort=3002, and timeZone=America/New_York in ".env" file. Then ran docker-compose up -d After logging into LibrePhotos, I entered the Nextcloud credentials with a new Nextcloud App Password and the Nextcloud Scan Directory status turned green. In the Admin Area, I see my Nextcoud directories listed and can select the any one of them. 
Now the Scan Photos (Nextcloud) button is darker and can finally scan my Nextcloud account directories.
